05-submitting_feature_requests.Rmd

```diff
@@ -0,0 +1,22 @@
+# Submitting Feature Requests {#FeatureRequests}
+
+## What is a feature request?
+
+A feature request is a suggestion or proposal to add new functionality or improve an existing feature.
+
+We recommend sharing your feature request idea on the [R-devel](https://stat.ethz.ch/mailman/listinfo/r-devel) mailing list and gaining support from R-Core members and other users to increase its likelihood of acceptance.
+
+## How to submit a feature request?
+
+You can submit a feature request by [filing a bug](https://bugs.r-project.org/enter_bug.cgi) on Bugzilla. Under Component, select "Wishlist", the designated label for feature requests.
+
+![The "Wishlist" component in Bugzilla's bug submission screen.](img/feature-requests.png)
+
+Similar to bugs, you should ensure that the feature request is [not already reported](https://contributor.r-project.org/rdevguide/ReportingBugs.html#checking-if-a-bug-is-already-reported) and [follow best practices](https://contributor.r-project.org/rdevguide/ReportingBugs.html#good-practices-in-reporting-bugs-expectations-of-a-good-bug-report) whenever possible.
```

Comment:
Please use the section to link instead of the full link (in case the section is moved somewhere else) like you did for the 03-issue_tracking.Rmd